symfony.lock

```diff
@@ -846,6 +849,9 @@
     "symfony/polyfill-php73": {
         "version": "v1.12.0"
     },
+    "symfony/polyfill-php80": {
+        "version": "v1.17.0"
+    },
```

Comment:
Just wondering, don't you know why polyfill for php8 is installed?

Reply:
I do not